Develop an SQL schema for SOPs

```diff
@@ -342,106 +480,203 @@ CREATE SEQUENCE public."PROTOCOLS_Protocol_ID_seq"
     CACHE 1;
 
 
-ALTER TABLE public."PROTOCOLS_Protocol_ID_seq" OWNER TO postgres;
+ALTER TABLE public."MFA_model_reactions_id_seq" OWNER TO postgres;
 
 --
--- TOC entry 2956 (class 0 OID 0)
--- Dependencies: 210
--- Name: PROTOCOLS_Protocol_ID_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+-- TOC entry 3455 (class 0 OID 0)
+-- Dependencies: 261
+-- Name: MFA_model_reactions_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
 --
 
-ALTER SEQUENCE public."PROTOCOLS_Protocol_ID_seq" OWNED BY public."PROTOCOLS"."Protocol_ID";
+ALTER SEQUENCE public."MFA_model_reactions_id_seq" OWNED BY public."MFA_model_reactions".id;
 
 
 --
--- TOC entry 213 (class 1259 OID 17528)
--- Name: PUBLICATIONS; Type: TABLE; Schema: public; Owner: postgres
+-- TOC entry 236 (class 1259 OID 18377)
+-- Name: MzTab-M_Metadata_metabolomics; Type: TABLE; Schema: public; Owner: postgres
 --
 
-CREATE TABLE public."PUBLICATIONS" (
-    "Publication_ID" integer NOT NULL,
-    "PubMed_ID" text,
-    "Publication_DOI" text,
-    "Publication_Author_List" text,
-    "Publication_Title" text,
-    "Publication_Status" text,
-    "Publication_Status_Term_Accession_Number" text,
-    "Publication_Status_Term_Source_REF" text
+CREATE TABLE public."MzTab-M_Metadata_metabolomics" (
+    id integer NOT NULL,
+    "mzTab-version" text NOT NULL,
```

Comment:
Please remove the `-` from all of the column names.  Similar to white spaces, this will cause all sorts of trouble during queries.

Reply:
done